content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -107,6 +108,31 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
+  IngressClass. Implementations can treat this as a separate PathType or treat
+  it identically to Prefix or Exact path types
+
+* **Exact:** Matches the URL path exactly and with case sensitivity.
+
+* **Prefix:** PathTypePrefix matches based on a URL path prefix split by '/'.
+  Matching is case sensitive and done on a path element by element basis. A path
+  element refers to the list of labels in the path split by the '/' separator. A
+  request is a match for path p if every p is an element-wise prefix of p of the
+  request path. Note that if the last element of the path is a substring of the
+  last element in request path, it is not a match (e.g. /foo/bar matches
+  /foo/bar/baz, but does not match /foo/barbaz).
```

Comment:
```suggestion
* **Prefix:** PathTypePrefix matches based on a URL path prefix split by `/`.
  Matching is case sensitive and done on a path element by element basis. A path
  element refers to the list of labels in the path split by the `/` separator. A
  request is a match for path _p_ if every _p_ is an element-wise prefix of _p_ of
  the request path.
    {{< note >}}
    If the last element of the path is a substring of the
    last element in request path, it is not a match (for example:
    `/foo/bar` matches`/foo/bar/baz`, but does not match `/foo/barbaz`).
    {{< /note >}}
```

(I think that'll render right, but please check!)

Reply:
Thanks, that is a lot better, updated.